web-client/src/presenter/state.ts

```diff
@@ -911,21 +911,22 @@ export type PractitionerDetail = {
   practitionerType?: string;
   middleName?: string;
   contact?: Partial<UserContact>;
-  openCaseInfo?: PractitionerCaseInfo; // Only for internal users
-  closedCaseInfo?: PractitionerCaseInfo; // Only for internal users
+  openCaseInfo?: PractitionerAllCasesInfo; // Only for internal users
+  closedCaseInfo?: PractitionerAllCasesInfo; // Only for internal users
```

Comment:
Renamed for clarity. Otherwise, `PractitionerCaseInfo` and `PractitionerCaseDetail` seem like the same thing.